docs/reference/tab-widgets/troubleshooting/disk/increase-data-node-capacity.asciidoc

```diff
@@ -0,0 +1,110 @@
+// tag::cloud[]
+In order to increase the disk capacity of the data nodes in your cluster:
+
+. Log in to the {ess-console}[{ecloud} console].
++
+. On the **Elasticsearch Service** panel, click the gear under the `Manage deployment` column that corresponds to the
+name of your deployment.
++
+. If autoscaling is available but not enabled, please enable it. You can do this by clicking the button
```

Comment:
All the instructions are based on the customer enabling autoscaling. Should we also offer some steps they can take if they are not willing/able to autoscale?

Reply:
For the sake of simplicity and to support we prefered to give only this option. We also believe that this is the way to scale in the cloud, we couldn't think of a reason why someone wouldn't be able to use autoscaling. Do you have such a use case?

Reply:
Autoscaling would be the preferred way, which might be the argument for mentioning only that option here (strategically). Many customers haven't enabled it, likely to due to concerns they would lose control over their spending or not yet fully embracing cloud mindset. OK to leave it as is for now. We'll be keeping our eye on the feedback for these docs, anyway.